pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -230,7 +230,8 @@ type FelixConfigurationSpec struct {
 	PrometheusGoMetricsEnabled *bool `json:"prometheusGoMetricsEnabled,omitempty"`
 	// PrometheusProcessMetricsEnabled disables process metrics collection, which the Prometheus client does by default, when
 	// set to false. This reduces the number of metrics reported, reducing Prometheus load. [Default: true]
-	PrometheusProcessMetricsEnabled *bool `json:"prometheusProcessMetricsEnabled,omitempty"`
+	PrometheusProcessMetricsEnabled   *bool `json:"prometheusProcessMetricsEnabled,omitempty"`
+	PrometheusWireGuardMetricsEnabled *bool `json:"prometheusWireGuardMetricsEnabled,omitempty"`
```

Comment:
You would need to add a comment so openapi-gen can pick it up with a description.